CHANGES.md

```diff
@@ -13,6 +13,7 @@ Unreleased changes refer to our current [master branch](https://github.com/openc
 * Fixed `Too many connections to ldap` when openchange runs on samba as member of a domain.
 * Fixed `Mark All as Read` feature regression bug introduced by 7737bdf6
 * Address book working much better than before
+* ResolveNames and ResoveNamesW operations perform a better Ambiguous Name Resolution search, which returns more matches.
```

Comment:
https://support.microsoft.com/en-us/kb/243299

The differences in our implementation:
- no RDN in samba
- proxyAddresses attribute: I haven't included them because one is not useful (admin mail) and the other two are in mailNickname and legacyDn
- if the string search fails the string is not splitted by whitespaces and the portions used for additional searches

The last one (string split by whitespaces) i see it useful but I have let it for other iteration. First I want this one in.